types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -1,5 +1,10 @@
+import {ANONYMOUS, SIGNED} from '../../../src/hooks/core/constant';
 import {ChannelList} from '../../database/schema/ChannelList.types';
 
+export interface ContactScreenPayload {
+  from: typeof ANONYMOUS | typeof SIGNED;
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `ContactScreenPayload` interface and its usage in `goToContactScreen` function is a good practice. It provides type safety and makes the code more predictable. However, it would be better to define the `from` property as an enumerated type instead of using `typeof`. This will make the code cleaner and easier to understand.
```diff
-  from: typeof ANONYMOUS | typeof SIGNED;
+  from: 'ANONYMOUS' | 'SIGNED';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->